test/test_helper.rb

```diff
@@ -120,7 +123,9 @@ def self.create(problem)
   def self.matrix_required(vrp)
     return unless ENV['TEST_DUMP_VRP'].to_s == 'true' && vrp.name
 
+    WebMock.enable_net_connect!
     vrp.compute_matrix
+    WebMock.disable_net_connect!
```

Comment:
This should allow us to define new tests and avoid locals `WebMock.enable_net_connect!`

Reply:
You tried to dump a tests and it works right? Do you know by any chance what `allow_net_connection` does by the way?